### DISCUSSION TITLE
Improve default player group names

### DISCUSSION THREAD

**Discussion by madbrain76** [music-assistant/support#2510] [Q&A] [answered]
When creating an Airplay player groups, and omitting the custom player name, the group shows up as "syncgroup_xxxxx".
For Universal Groups, they show up as ugp_xxxxxx".

These names are not very user friendly. Also, they are assigned only after clicking "save", so the user doesn't know the name of the group that's just been created.

I'm proposing that :
a) the default group name should be more intuitive to the user, perhaps a name derived from the member player names., something like "airplay_group_member1_member2" .
b) the default name should be shown to the user one way or another before clicking save

**Comment by marcelveldt** [comment]
I have no idea what you mean here, you can give your own name to a group player, just like any other player. It will show in the list with that name.

That ugp prefixed tag is just the player id, not the name

**Comment by madbrain76** [comment]
@marcelveldt ,

1. as far as player id vs name - the confusion is due to the fact that the settings/players list shows them in the same place.

For example, if I enter a custom player name, I get this :
![image](https://github.com/music-assistant/support/assets/57333073/e91a940f-2234-4a62-b985-bab1d45fb82f)
And if I don't, I get this :
![image](https://github.com/music-assistant/support/assets/57333073/1f4992d8-4903-48a7-9a0a-4d23762c6ba1)

These strings are displayed in the same locations, which is why I thought they were both names. But it looks like the list shows the name if there is one, and shows the id if there isn't one. But the user doesn't know one is a name and the other is an ID. My impression was that the later was a default name, not an ID. That is a technicality, though.

2. The problem I'm trying to solve is "where the hell is my new group player in the player list" after clicking save without assigning a custom player name. If you only have a few players, it will be easy to find the new group in the list. But if you have 48, and have no idea what the id is going to be, then it's not as obvious, especially the first time you do it.

The non-group players are typically automatically discovered, and thus entering a name for them is not mandatory, and it is not hard to find them because the user already knows the device names, as they have previously been set in external applications.

**Comment by OzGav** [comment, marked as answer]
Just looping back on this for future readers. The docs are now clear that a custom name needs to be added when setting up the group https://www.music-assistant.io/faq/groups/

> After selecting ADD GROUP PLAYER as shown above, the group configuration screen will appear where the group type must be selected. You need to specify a name for the group player. The order that the players are selected when the group is created will be the order that the players are shown when viewing the group in the [Player List](https://www.music-assistant.io/ui/#player-list).
